CMakeLists.txt

```diff
@@ -286,7 +286,9 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Clang"
     target_compile_options(ds2 PRIVATE -Wsuggest-override)
   endif ()
   if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    target_compile_options(ds2 PRIVATE -Winconsistent-missing-override)
+    if (NOT "${OS_NAME}" MATCHES "FreeBSD")
```

Comment:
true. but I don't know if it's specific to freebsd (incomplete port of clang ?) or due to a specific version. I did a quick search but didn't found any page with feature/version. Will search again.